Extension/package.json

```diff
@@ -3309,67 +3309,116 @@
+      ],
+      "integrity": "07BCB0E787B440D55377DA475BC6859694C5EDDFE4540DB11D5EC864B67C29E3"
     },
     {
-      "description": "Mono Runtime (Linux / x86_64)",
-      "url": "https://go.microsoft.com/fwlink/?LinkId=2027416",
+      "description": "OpenDebugAD7 (Windows ARM64)",
+      "url": "https://download.visualstudio.microsoft.com/download/pr/65d2e4a8-b9e4-40e8-a131-51b3b6af7b43/c6fc1101fc22085d80ee5b51b05cf75b/win10-arm64.zip",
+      "platforms": [
+        "win32"
+      ],
+      "architectures": [
+        "arm64"
+      ],
+      "binaries": [
+        "./debugAdapters/bin/OpenDebugAD7.exe"
+      ],
+      "integrity": "98E7434DA74545FE0731E2BB08A14B92B075A0A888181223E45221EA823EE42C"
+    },
+    {
+      "description": "OpenDebugAD7 (Linux x64)",
+      "url": "https://download.visualstudio.microsoft.com/download/pr/65d2e4a8-b9e4-40e8-a131-51b3b6af7b43/cab96f290df15021d58396616c833fa4/linux-x64.zip",
       "platforms": [
         "linux"
       ],
       "architectures": [
         "x64"
       ],
       "binaries": [
-        "./debugAdapters/mono.linux-x86_64"
+        "./debugAdapters/bin/OpenDebugAD7"
       ],
-      "integrity": "927C5094E83CE64EDC3A267EE1F13267CFA09FC1E9A5ADC870A752C05AD26F17"
+      "integrity": "4912413127724649A10A0925316D7E735159F49405D7DD3EA2F54750D6F65964"
     },
     {
-      "description": "Mono Runtime (Linux / armhf)",
-      "url": "https://aka.ms/mono-runtime-linux-arm",
+      "description": "OpenDebugAD7 (Linux ARM)",
+      "url": "https://download.visualstudio.microsoft.com/download/pr/65d2e4a8-b9e4-40e8-a131-51b3b6af7b43/3aaff9a6621469d7efd1423fe2c1063e/linux-arm.zip",
       "platforms": [
         "linux"
       ],
       "architectures": [
         "arm"
       ],
       "binaries": [
-        "./debugAdapters/mono.linux-armhf"
+        "./debugAdapters/bin/OpenDebugAD7"
       ],
-      "integrity": "D2D1FE7FB5CC5B2A60364F08829467509879D603FC951AC3805B530C8CEEF981"
+      "integrity": "527426D22FCDD0F3A679F767DDEE82BE34A73A281A46FF3CD64771D37EF6B886"
     },
     {
-      "description": "Mono Runtime (Linux / arm64)",
-      "url": "https://aka.ms/mono-runtime-linux-arm64",
+      "description": "OpenDebugAD7 (Linux ARM64)",
+      "url": "https://download.visualstudio.microsoft.com/download/pr/65d2e4a8-b9e4-40e8-a131-51b3b6af7b43/3ab22dae7df3d59330d73ce58c9025b5/linux-arm64.zip",
       "platforms": [
         "linux"
       ],
       "architectures": [
         "arm64"
       ],
       "binaries": [
-        "./debugAdapters/mono.linux-arm64"
+        "./debugAdapters/bin/OpenDebugAD7"
       ],
-      "integrity": "946C54C8C6BF5BF79AC05D4E152F8D8647700FA786C21505832B3C79988339B4"
+      "integrity": "2EB2F631C70AB4C6717D5AC61009051F6EFD4BFAF68E7364D95533CA2A7F2F03"
     },
     {
-      "description": "Mono Runtime (macOS)",
-      "url": "https://go.microsoft.com/fwlink/?LinkId=2027403",
+      "description": "OpenDebugAD7 (Alpine Linux x64)",
```

Comment:
Should we disable the "Alpine containers are not supported." message and instead just disable the language service stuff (as if intelliSenseEngine was "Disabled") so this can be used?

Reply:
I was not sure if we wanted to ship without language service. 

Theres a couple issues with the package manager needing to detect Alpine Linux and the ConfigurationProvider seems to be integrated with the language service. I will need to either remove the configuration provider during alpine launch or refactor so it does not use the language service in that scenario.

Reply:
Now or later?

Reply:
I'll do it in another check-in. For now, lets get our currently supported platforms updated.